src/frontend/src/index.css

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Prompt:ital,wght@1,800&display=swap');
```

Comment:
We should consider downloading the font files later on.

Reply:
Can do